packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -5811,6 +5875,8 @@ abstract class ComponentElement extends Element {
   @override
   @pragma('vm:notify-debugger-on-exception')
   void performRebuild() {
+    _currentBuildDependencies?.clear();
```

Comment:
Why do we only call this `_currentBuildDependencies?.clear()` and `cleanupRemovedDependencies()` in `ComponentElement`? Why doesn't `Element` need it? If `Element` doesn't need it, can we move everything related to `_currentBuildDependencies` to `ComponentElement`?

Reply:
I don't think at Element, we are handling dependencies. I have moved _currentBuildDependencies to `ComponentElement`.